### ISSUE TITLE
"handle is invalid" on terminal command, powershell

### ISSUE THREAD

**Issue by Productive2** [inim-repl/INim#115]
How to reproduce this bug?

1. Open powershell
2. Install inim via `nimble install inim` or clone the master branch and build by `nimble build`, it can be reproduced both ways
3. `import terminal`
4. `setCursorXPos(1)`

Outcome:

![image](https://user-images.githubusercontent.com/48047721/108603632-55bc4080-73a9-11eb-9074-850366235ad6.png)

Expected Outcome:

Not sure, but definitely not an error message.

Platform: windows 10, powershell,  inim 0.6.1

**Comment by Productive2** [comment]
Also happens with setCursorPos, eraseScreen, eraseLine, cursorUp...

**Comment by 0atman** [comment]
I don't have a windows machine to test this on, can anyone help here?

**Comment by Productive2** [comment]
> Can't replicate this on Ubuntu, and I don't have a windows machine to test this on, can anyone help here?

Yes, I also tried on linux, powershell is the only platform I could replicate it on.
